ROLES.md

```diff
@@ -58,22 +57,55 @@ table describes:
     </td>
     <td>Highly experienced and active reviewer and contributor to an area</td>
     <td>Entry in one or more OWNERS files in GitHub, and write permissions
-        on one or more repos allowing PRs to be merged.
+        on one or more repos allowing PRs to be merged
     </td>
     <td>GitHub Directory</td>
   </tr>
 
   <tr>
-    <td>Lead</td>
+    <td>Working Group Technical Lead</td>
     <td>
         <p>Set priorities for a functional area and approve proposals</p>
         <p>Triage incoming issues, set milestones, repo labels</p>
-        <p>Run their working group</p>
+        <p>Roadmap alignment with top-level backlog</p>
+        <p>Mentor new contributors, project members, and approvers</p>
+        <p>Succession - identifying next steps for members of the working group</p>
+        <p>Responsible for technical health of their functional area</p>
     </td>
     <td>Sponsored by the technical oversight committee as documented
-        <a href="./mechanics/WORKING-GROUP-PROCESSES.md">here</a>.
+        <a href="./mechanics/WORKING-GROUP-PROCESSES.md">here</a>
     </td>
-    <td>Write permissions on one or more repos allowing issues to be manipulated.</td>
+    <td>Write permissions on one or more repos allowing issues to be manipulated</td>
+    <td>Working Group</td>
+  </tr>
+
+  <tr>
+    <td>Working Group Execution Lead</td>
+    <td>
+        <p>Run their working group: Meetings, notes, roadmap, report</p>
+        <p>Responsible for the holistic health of the <i>working group</i></p>
+        <p>Work organization, planning, high-level execution</p>
+        <p>Triage incoming issues, set milestones, repo labels</p>
+        <p>Ensure all required skills are present within the working group</p>
+        <p>Mentor new contributors, project members, and approvers</p>
+    </td>
+    <td>Sponsored by the technical oversight committee as documented
+        <a href="./mechanics/WORKING-GROUP-PROCESSES.md">here</a>
+    </td>
+    <td>Write permissions on one or more repos allowing issues to be manipulated</td>
+    <td>Working Group</td>
+  </tr>
+
+  <tr>
+    <td>Scribe</td>
+    <td>
+        <p>Ensure important information is represented in working group notes</p>
+        <p>Post WG recordings to team drive</p>
+    <td>
+    <td>Sponsored by the technical oversight committee as documented
```

Comment:
Is this "sponsored by TOC" rather than "selected by WG leads" because we don't have a good way to grant WG leads that permission?

(I only ask because I'd be happy to see the WG leads do this themselves, but I know that the calendar stuff is touchy with the current Google Calendar instance.)

Reply:
It's a typo, fixed